### PR TITLE
Init wcm-proxy-helper

### PR DESCRIPTION
**Pull request by JvJefke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
...
+
+```javascript
+var ProxyHelper = require("wcm-proxy-helper");
+
+module.exports = function(app) {
+    // Setup proxy for WCM
+    ProxyHelper(app, {
+        target: "https://target...",
+        apikey: "WCM apikey",
+        tenant: "WCM tenant name,
```
Comment (ThomasBormans) [inline README.md]: Missing quote

File: README.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
+
+### Options
+
+| Option | Default | Required | Description |
+|--------|---------|----------|-------------|
+| target | undefined | true | Target to proxy to. |
+| apikey | undefined | false | WCM apikey |
+| tenant | undefined | false | WCM tenant |
+| host | undefined | false | Server host. (This needs to be set, when setting up a proxy from http to https) |
+| prefix | "/proxy" | false | Route after which should be proxied |
```
Comment (ThomasBormans) [inline README.md]: Maybe mention if a trailing/leading slash is required?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,54 @@
...
+`ProxyHelper(app, options);`
+
+### Options
+
+| Option | Default | Required | Description |
+|--------|---------|----------|-------------|
+| target | undefined | true | Target to proxy to. |
+| apikey | undefined | false | WCM apikey |
+| tenant | undefined | false | WCM tenant |
+| host | undefined | false | Server host. (This needs to be set, when setting up a proxy from http to https) |
```
Comment (ThomasBormans) [inline README.md]: Maybe mention if a trailing/leading slash is required?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,54 @@
...
+
+## ProxyHelper
+
+`ProxyHelper(app, options);`
+
+### Options
+
+| Option | Default | Required | Description |
+|--------|---------|----------|-------------|
+| target | undefined | true | Target to proxy to. |
```
Comment (ThomasBormans) [inline README.md]: Maybe mention if a trailing/leading slash is required?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,54 @@
...
+## addProxyRoute
+
+'ProxyHelper.addProxyRoute(app, prefix, options);'
+
+### Options
+
+| Option | Default | Required | Description |
+|--------|---------|----------|-------------|
+| tenant | undefined | false | WCM tenant |
+| host | undefined | false | Server host. (This needs to be set, when setting up a proxy from http to https). |
```
Comment (ThomasBormans) [inline README.md]: Maybe mention if a trailing/leading slash is required?

File: README.md [hunk 6]
```diff
@@ -1,2 +1,54 @@
...
+| headers | undefined | false | Addtional headers |
```
Comment (ThomasBormans) [inline README.md]: Add an empty line at the end of the file.

File: .eslint/defaults.js [hunk 7]
```diff
@@ -0,0 +1,305 @@
...
+		// Not needed with "impliedStrict"
+		// "strict": [2, "global"], // require or disallow strict mode directives
+
+
+		////////// Variables //////////
+
+		"no-catch-shadow": 0,             // disallow the catch clause parameter name being the same as a variable in the outer scope (off by default in the node environment)
+		"no-delete-var": 0,               // disallow deletion of variables
+		"no-label-var": 0,                // disallow labels that share a name with a variable
+		"no-shadow": 1,                   // disallow declaration of variables already declared in the outer scope
```
Comment (ThomasBormans) [inline .eslint/defaults.js]: This should equal _error_ instead of _1_.

File: .eslint/defaults.js [hunk 8]
```diff
@@ -0,0 +1,305 @@
...
+		"no-shadow-restricted-names": 0,  // disallow shadowing of names such as arguments
+		"no-undef": 2,                    // disallow use of undeclared variables unless mentioned in a /*global */ block
+		"no-undef-init": 0,               // disallow use of undefined when initializing variables
+		"no-undefined": 0,                // disallow use of undefined variable (off by default)
+		"no-unused-vars": ["warn"],       // disallow declaration of variables that are not used in the code
```
Comment (ThomasBormans) [inline .eslint/defaults.js]: Please remove the "warn" message and throw an error: _["error", { "args": "after-used" }]_.

File: package.json [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "WCM",
+    "Pelorus",
+    "Proxy"
+  ],
+  "author": "Jeroen Valcke (District01)",
```
Comment (ThomasBormans) [inline package.json]: It is better to enter your name only in the _author_ field. Add a _contributators_ field for multiple contributors and more information: [docs](https://docs.npmjs.com/files/package.json).

File: package.json [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+  "license": "ISC",
```
Comment (ThomasBormans) [inline package.json]: We are using the _GPL-2.0_ license for Hyperdrive.

File: package.json [hunk 11]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "wcm-proxy-helper",
+  "version": "0.0.1",
+  "description": "Proxy helper for WCM node server",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (ThomasBormans) [inline package.json]: 👀

File: index.js [hunk 12]
```diff
@@ -0,0 +1,92 @@
+"use strict";
+
+require("rootpath")();
```
Comment (ThomasBormans) [inline index.js]: I do not think you need rootpath because you are only requiring node_modules.

File: README.md (hunk 1 above)
Comment (JvJefke) [inline README.md]: fixed

File: README.md (hunk 2 above)
Comment (JvJefke) [inline README.md]: Updated the code so this shouldn't matter

File: README.md (hunk 3 above)
Comment (JvJefke) [inline README.md]: Updated the code so this shouldn't matter

File: README.md (hunk 4 above)
Comment (JvJefke) [inline README.md]: Updated the code so this shouldn't matter

File: README.md (hunk 5 above)
Comment (JvJefke) [inline README.md]: Updated the code so this shouldn't matter

File: README.md (hunk 6 above)
Comment (JvJefke) [inline README.md]: done

File: .eslint/defaults.js (hunk 7 above)
Comment (JvJefke) [inline .eslint/defaults.js]: Changed to FEO eslint

File: .eslint/defaults.js (hunk 8 above)
Comment (JvJefke) [inline .eslint/defaults.js]: Changed to FEO eslint

File: package.json (hunk 9 above)
Comment (JvJefke) [inline package.json]: Changed conform with node boilerplate

File: package.json (hunk 10 above)
Comment (JvJefke) [inline package.json]: Fixed

File: package.json (hunk 11 above)
Comment (JvJefke) [inline package.json]: Yeah I know. This is the init. Tests are coming in the next lounge.

File: index.js (hunk 12 above)
Comment (JvJefke) [inline index.js]: You are correct! fixed...

File: package.json (hunk 11 above)
Comment (JvJefke) [inline package.json]: @ThomasBormans Tests are added.

**Comment by ThomasBormans** [comment]
Kudos @JvJefke! 🎉

**Pull request merged**